tests/pytorch/nightly/hf-glue.libsonnet

```diff
@@ -167,7 +163,7 @@ local utils = import "templates/utils.libsonnet";
       ||| % {common: command_common, common_copy: command_copy_metrics}
     ),
     regressionTestConfig+: {
-      alert_for_failed_jobs: false,
+      required_metrics: ['eval_mnli/acc'],
```

Comment:
@jysohn23 let me know which metrics are good to include here

Reply:
Let's not include these yet since those metrics haven't been renamed yet and I'm still looking at a fix for the regression. Let's add then once I get to HF fix.